### DISCUSSION TITLE
问号位置和方向有误

### DISCUSSION THREAD

**Discussion by A6669** [Ancient-China-Books/guwenguanzhi#3] [「卷一 周文」鄭伯克段于鄢]
https://ancient-china-books.github.io/guwenguanzhi/text/part005.html
在上述文章中，很多问号位置不对，例如“姜氏何厭（平聲。）之有（厭，足也。）？” 应为“姜氏何厭（平聲。）之有？（厭，足也。）”
有些问号方向不对，例如“潁考叔曰：「敢問何謂也佯為不知。妙。? ” 在竖排显示中 ？方向不对
望修正

**Comment by douglarek** [comment]
> 敢問何謂也佯為不知。妙。? ” 在竖排显示中 ？方向不对

這個確實，非常感謝，這個地方其實是原電子書混合了中英文問號。

![image](https://user-images.githubusercontent.com/1488134/172032496-da793832-f592-4ac4-8f23-86c39a11ac9c.png)

**Reply by A6669** [reply]
某些问号位置不对，例如“姜氏何厭（平聲。）之有（厭，足也。）？” 应为“姜氏何厭（平聲。）之有？（厭，足也。）”

**Reply by douglarek** [reply]
> 某些问号位置不对，例如“姜氏何厭（平聲。）之有（厭，足也。）？” 应为“姜氏何厭（平聲。）之有？（厭，足也。）”

是的，我也發現了這個問題，夾註我當時是用腳本轉的，記不太清是我腳本轉換的問題還是它原電子書鏈接錯位的問題，不過還好，改正起來是個體力活，我已經改了一版。應該其他篇章也有這個問題，不過這個改起來體力活比較大（私以爲不影響原文正確性，可以優先級調低），可以優先改錯位的符號以及錯別字。

**Reply by douglarek** [reply]
已修復：https://github.com/Ancient-China-Books/guwenguanzhi/commit/2180aa593f5acc204defeaf3f0b5a836925e44b4 ，可以重新訪問：https://ancient-china-books.github.io/guwenguanzhi/text/part005.html 查看或者下載最新的 epub。
